src/components/Post/GovernanceSideBar/PIPs/PIPsVote.tsx

```
@@ -57,7 +57,7 @@ export const getConvictionVoteOptions = (CONVICTIONS: [number, number][], propos
 	if ([ProposalType.REFERENDUM_V2, ProposalType.FELLOWSHIP_REFERENDUMS].includes(proposalType) && ![AllNetworks.COLLECTIVES, AllNetworks.WESTENDCOLLECTIVES].includes(network)) {
 		if (api && apiReady) {
 			const res = api?.consts?.convictionVoting?.voteLockingPeriod;
-			const num = res.toJSON();
+			const num = res?.toJSON();
 			const days = blockToDays(num, network);
 			if (days && !isNaN(Number(days))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)